lib/floe/workflow/intrinsic_function/transformer.rb

```diff
@@ -151,6 +151,11 @@ def check_types(args, function, signature)
           OpenSSL::Digest.hexdigest(algorithm, data)
         end
 
+        rule(:states_json_to_string => {:args => subtree(:args)}) do
+          args = Transformer.process_args(args(), "States.JsonToString", [Object])
+          JSON.generate(args.first)
```

Comment:
Is it possible for JSON.generate to fail?  It feels like it could, but I'm not sure how.  If it can fail with a specific error, we will likely need to rescue and wrap that in a Floe error.